scripts/run.sh

```diff
@@ -228,6 +229,46 @@ function update() {
         dotnet Setup.dll -update 1 -os $OS -corev $COREVERSION -webv $WEBVERSION -keyconnectorv $KEYCONNECTORVERSION
 }
 
+function uninstall() {
+    echo -e -n "${RED}(WARNING: UNINSTALL STARTED) Would you like to save the database files? (y/n): ${NC}"
+    read KEEP_DATABASE
+
+    if [ "$KEEP_DATABASE" == "y" ]
+    then
+        echo "Saving database files."
+        zip -r "./bitwarden_database.zip" "$OUTPUT_DIR/mssql"
```

Comment:
```suggestion
        tar -cvzf "./bitwarden_database.tar.gz" "$OUTPUT_DIR/mssql"
```

Reply:
Do most Linux distros include zip by default? I think tar might be more popular. Thoughts?